### ISSUE TITLE
Does not find all games, even when on IGDB

### ISSUE THREAD

**Issue by GirafiStudios** [tauqua/gog-galaxy-itch.io#3]
For example the game "No Delivery" is on IGDB, but shows up as Unknown Game on GOG.

Not sure if it checks the exact title before it grabs information about the game from IGDB, or how it all works - would love to know!

**Comment by tauqua** [comment]
GOG maintains their own database, accessible at https://gamesdb.gog.com/platforms/{platform}/external_releases/{id}. While it seems to draw from IGDB, I'm not positive how it's updated and I'm not aware of a way to make contributions to it, hence directing people to IGDB. 

No Delivery should show up on that database at [https://gamesdb.gog.com/platforms/itch/external_releases/429778](https://gamesdb.gog.com/platforms/itch/external_releases/429778), where the game id is found from an itch API search [https://api.itch.io/search/games?query=no+delivery](https://api.itch.io/search/games?query=no+delivery) or looking in the plugin log file, which lists the name and Itch ID of every game it attempts to import.

If you want to make sure that the plugin is grabbing the correct name, you can check the log at the location described [here](https://github.com/Mixaill/awesome-gog-galaxy#installation-and-debugging), where it should say "Parsed No Delivery, 429778". 

In summary, the end all database is run by GOG without any method for community contribution as far as I know. If you're curious about whether a game is in GOG's database, check the log file for the "Parsed" line to find it's id number, then go to https://gamesdb.gog.com/platforms/itch/external_releases/{id} and see if a listing exists.
